website/versioned_docs/version-22.3/SnapshotTesting.md

```diff
@@ -158,17 +189,17 @@ Now, every time the snapshot test case runs, `Date.now()` will return
 `1482363367071` consistently. This will result in the same snapshot being
 generated for this component regardless of when the test is run.
 
-### Snapshots are not written automatically on Continuous Integration systems (CI)
```

Comment:
It was phrased like this because of SEO. I guess Google should handle this though